code-of-conduct.md

```diff
@@ -33,7 +33,19 @@ Participants asked to stop any harassing behavior are expected to comply immedia
 
 ## Scope
 
-This Code of Conduct applies both at the event and in public spaces when an individual is representing Our Networks.
+This Code of Conduct applies at the event, in the online spaces of the conference, and in public spaces when an individual is representing Our Networks.
+
+## Content Moderation Guidelines
+
+Because of ensuring this code of conduct is enforcable in an online context across platforms, we have some guidelines about the types of spaces the conference includes.
+
+* **Moderated** - one of the core team or a trained volunteer is present for the full duration of the event actively moderating. The platform provides at least a minimum set of tools for moderation.
+
+* **Partially moderated** - either the space is durational and cannot be continuously moderated (but a member of the core team or trained volunteer will check in periodically to check on participants and moderate if needed) and/or the space has some moderation tools but they may not be ideal or "complete"
+
+* **Unmoderated** - the space cannot be moderated because of the nature of the platform/artwork/etc please use these spaces at your own risk/discretion, we will do our best to clearly mark them
```

Comment:
```suggestion
* **Unmoderated** - the space cannot be moderated because of the nature of the platform/artwork/etc please use these spaces at your own risk/discretion, we will do our best to clearly mark them.
```